aunt-leahs-app/src/redux/volunteer/volunteerAction.js

```diff
@@ -11,3 +11,10 @@ export const setExistingVolunteer = existingVolunteer => (
         payload: existingVolunteer
     }
 )
+
+export const setDuration = duration =>(
+    {
+        type: 'SET_DURATION',
+        payload: duration
+    }
+)
```

Comment:
Add a space after `=>` and do `({` and `})` together

Reply:
no